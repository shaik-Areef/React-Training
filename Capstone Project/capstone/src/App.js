
import { Route, HashRouter as Router, Routes } from 'react-router-dom';
import Navigationbar from './Components/NavigationBar/Navigationbar';
import AboutUs from './Pages/AboutPage/AboutUs/AboutUs';
import ContactUs from './Pages/ContactPage/ContactUs/ContactUs';
import Products from './Pages/ProductPage/Products/Products';
import ProductDetails from './Components/ProductDetails/ProductDetails';
import TabProducts from './Components/TabProucts/Tabproducts';

function App() {
  return (
    <div className="App">
      <Router>
        <Navigationbar />
        <Routes>
          <Route exact path='/' element={<TabProducts/>} />
          <Route exact path='/products/product-category' element={<Products />} />
          <Route exact path='/products/product-category/:id' element={<ProductDetails />} />
          <Route exact path='/about-us' element={<AboutUs />} />
          <Route exact path='/contact-us' element={<ContactUs />} />
        </Routes>
      </Router>
    {/* <GetInTouch/> */}
      
    </div>
  );
}

export default App;
