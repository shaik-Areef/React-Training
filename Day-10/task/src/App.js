//import logo from './logo.svg';
//import './App.css';
import Webform from "./WebForm";

function App() {
  return (
    <div className="App">
      <Webform></Webform>
    </div>
  );
}

export default App;
