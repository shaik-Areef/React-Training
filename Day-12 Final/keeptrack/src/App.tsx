import React from 'react';
import logo from './logo.svg';
// import './App.css';
import ProjectsPage from './Projects/Projects_page';

function App() {
  return (
    <div className="App">
      <ProjectsPage/>
    </div>
  );
}

export default App;
