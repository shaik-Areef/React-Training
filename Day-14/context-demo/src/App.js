import React from "react";
import Content from "./Content";
import ContactUsForm from "./FormComponent"
import ExampleForm from "./Uncontorlled";
const App = () => {
  return (
    <div>
      {/* <Content /> */}
      {/* <ContactUsForm/> */}
      <ExampleForm/>
    </div>
  )
}
export default App;
