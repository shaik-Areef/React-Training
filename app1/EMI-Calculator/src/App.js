//import './App.css';
import EmiCalculator from "./EMI-Calculator";
function App() {
  return (
    <div className="App">
      <header className="App-header">
      <EmiCalculator></EmiCalculator>
      </header>
    </div>
  );
}

export default App;
