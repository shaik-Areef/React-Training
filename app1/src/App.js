
import OrderForm from './orderform';

function App() {
  return (
    <div className="App">
      <header className="App-header">
      <OrderForm name='Ramesh'></OrderForm>
      </header>
    </div>
  );
}

export default App;
