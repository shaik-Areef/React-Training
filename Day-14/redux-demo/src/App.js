import logo from './logo.svg';
import './App.css';
import LogState from './Counter';

function App() {
  return (
    <div className="App">
      <LogState></LogState>
    </div>
  );
}

export default App;
