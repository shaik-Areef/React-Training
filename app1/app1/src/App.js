import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
      <OrderForm name='Ramesh'></OrderForm>
      </header>
    </div>
  );
}

export default App;
