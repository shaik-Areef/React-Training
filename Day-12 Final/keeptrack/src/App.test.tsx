import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';
import ProjectsPage from './Projects/Projects_page';
import renderer from 'react-test-renderer';



describe('<Projectspage />', () => {
  test('should render without crashing', () => {
    render(<App />);
  });

  test('snapshot', () => {
    const tree = renderer.create(<ProjectsPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});


