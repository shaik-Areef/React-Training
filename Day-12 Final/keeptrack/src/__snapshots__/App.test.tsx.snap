// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Projectspage /> snapshot 1`] = `
Array [
  <h1>
    Projects
  </h1>,
  <div
    className="row"
  />,
]
`;
